core/src/test/scala/com/pingcap/tispark/auth/TiAuthIntegrationSuite.scala

```diff
@@ -94,12 +94,7 @@ class TiAuthIntegrationSuite extends SharedSQLContext {
   test("Use database and select without privilege should not be passed") {
     the[SQLException] thrownBy spark.sql(
       f"use $databaseWithPrefix") should have message s"Access denied for user $user@% to database ${databaseWithPrefix}"
-    if (catalogPluginMode) {
-      an[AnalysisException] should be thrownBy spark.sql(s"select * from $table")
-    } else {
-      the[SQLException] thrownBy spark.sql(
-        s"select * from $table") should have message s"SELECT command denied to user $user@% for table $databaseWithPrefix.$table"
-    }
+    an[AnalysisException] should be thrownBy spark.sql(s"select * from $table")
```

Comment:
Can we check the message of AnalysisException

Reply:
good idea